src/App.css

```diff
@@ -205,7 +205,7 @@ nav {
   animation-delay: 0.4s;
 }
 
-.list__item {
+li {
```

Comment:
Also, please, could you revert this to .list__item?

Reply:
Changed it because I need to assign new classNames to the list item, and because I'm passing it in as a variable, that complicates things